Frontend/ui-customer/index.js

```diff
@@ -113,10 +113,12 @@ window.onMerchantEvent = function (event, payload) {
     {
       clientId = clientId.replace("_ios","");
     }
-    if (clientId == "open-kochi") {
+    if (clientId == "open-kochi" || clientId == "yatriconsumer") {
```

Comment:
as we removed `open-kochi` from everywhere can we remove it from index.js file as well and refactor accordingly?

Reply:
Can we keep till we completely migrate to yatriconsumer and later remove

Reply:
client-id is been set in initiate payload of hyper-sdk , if we remove open-kochi clientId condition  and push the bundle it will break